### ISSUE TITLE
Crash after putting the timer at the end of the song

### ISSUE THREAD

**Issue by jja2000** [naman14/Timber#164]
Hey, when I pull the timer all the way to the end of the song and hold it there it crashes the app. (It seems to be reproduceable tell me if you want a logcat)

**Comment by naman14** [comment]
Timer as in slider? Was unable to reproduce it on my device. Please share logcat

**Comment by jja2000** [comment]
Yes! The slider (forgot the word, English is not my native language)
[Here it is](https://github.com/naman14/Timber/files/165612/timber-slidercrash-logcat.txt)

**Comment by naman14** [comment]
No problem:) Looking into the crash
